python/cuml/cuml/cluster/hdbscan/hdbscan.pyx

```diff
@@ -42,7 +42,7 @@ IF GPUBUILD == 1:
     from libc.stdlib cimport free
     from cython.operator cimport dereference as deref
     from cuml.metrics.distance_type cimport DistanceType
-    from rmm._lib.device_uvector cimport device_uvector
+    from rmm.librmm.device_uvector cimport device_uvector
```

Comment:
We will need to add pylibrmm and librmm to dependencies.yaml, right?

Reply:
No `librmm` and `pylibrmm` are just subpackages of rmm, which should already be a dependency.

Reply:
Does cuML uses RMM nightlies?